<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>627</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Генератор игр v0.01</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="genre_fr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Жанр:</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>192</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="genre">
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>FPS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Платформер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>RTS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>Rouge-Like</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>Мультиплеер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>RPG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="text">
         <string>Battle Royale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>151</width>
       <height>192</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="genre_2">
      <item>
       <widget class="QCheckBox" name="checkBox_13">
        <property name="text">
         <string>Хоррор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_14">
        <property name="text">
         <string>Аркада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_15">
        <property name="text">
         <string>Extraction Shooter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_16">
        <property name="text">
         <string>Hack and Slash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_17">
        <property name="text">
         <string>Головоломка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_18">
        <property name="text">
         <string>Gacha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_19">
        <property name="text">
         <string>Симулятор</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="optimization">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>53</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="optimization_l">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QSlider" name="optimization_slider">
        <property name="maximum">
         <number>4</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TickPosition::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="percent_l">
        <item>
         <widget class="QLabel" name="zero">
          <property name="text">
           <string>0%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fifty">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;50%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="one_hundred">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;100%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Оптимизация</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="features_fr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <widget class="QLabel" name="features_l">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Фичи:</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>172</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="features">
      <item>
       <widget class="QCheckBox" name="checkBox_10">
        <property name="text">
         <string>Крутая физика</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_11">
        <property name="text">
         <string>Крутая графика</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>Raytracing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_9">
        <property name="text">
         <string>DLSS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_12">
        <property name="text">
         <string>Можно грабить корованы</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>201</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="features_2">
      <item>
       <widget class="QCheckBox" name="checkBox_20">
        <property name="text">
         <string>Много скинов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_21">
        <property name="text">
         <string>AAAA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_22">
        <property name="text">
         <string>Solus-like (типа очень сложна)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_23">
        <property name="text">
         <string>Огромные анимешные сиськи</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_24">
        <property name="text">
         <string>Инклюзивность</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="game_name">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Название игры:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="game_save">
     <item>
      <widget class="QLabel" name="save_to_folder">
       <property name="text">
        <string>Сохранить в папку:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path"/>
     </item>
     <item>
      <widget class="QPushButton" name="file_picker">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="create_b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255))</string>
    </property>
    <property name="text">
     <string>СОЗДАТЬ!!!</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>:P</string>
    </property>
    <addaction name="token_settings"/>
   </widget>
   <addaction name="menu_P"/>
  </widget>
  <action name="token_settings">
   <property name="text">
    <string>Настройки токенов</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>optimization_slider</tabstop>
  <tabstop>checkBox_10</tabstop>
  <tabstop>checkBox_11</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>checkBox_9</tabstop>
  <tabstop>checkBox_12</tabstop>
  <tabstop>path</tabstop>
  <tabstop>file_picker</tabstop>
  <tabstop>create_b</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
